src/main.c

```diff
@@ -92,21 +80,37 @@
 #define NEXT_CONN_PARAMS_UPDATE_DELAY   APP_TIMER_TICKS(5000)                   /**< Time between each call to sd_ble_gap_conn_param_update after the first call (5 seconds). */
 #define MAX_CONN_PARAMS_UPDATE_COUNT    3                                       /**< Number of attempts before giving up the connection parameter negotiation. */
 
-#define BUTTON_DETECTION_DELAY          APP_TIMER_TICKS(50)                     /**< Delay from a GPIOTE event until a button is reported as pushed (in number of timer ticks). */
+// #define TX_POWER_LEVEL -8
+// #define TX_POWER_LEVEL 0
+// #define TX_POWER_LEVEL 4
+#define TX_POWER_LEVEL 8
+
+// #define BUTTON_DETECTION_DELAY          APP_TIMER_TICKS(50)                     /**< Delay from a GPIOTE event until a button is reported as pushed (in number of timer ticks). */
+
+// #define DEAD_BEEF                       0xDEADBEEF                              /**< Value used as error code on stack dump, can be used to identify stack location on stack unwind. */
+
+#define UUID_BASE {0x23, 0xD1, 0xBC, 0xEA, 0x5F, 0x78, 0x23, 0x15, 0xDE, 0xEF, 0x12, 0x12, 0x00, 0x00, 0x00, 0x00}
+#define UUID_SERVICE 0x1234
```

Comment:
Should this have the same UUID as UUID_VOLTAGE_CHAR on the next line?

Reply:
The UUIDs are not delimiters, they are used to define the ID of the service and the characteristics. So, I randomly picked the hex values for the UUIDs. I am pretty sure it has no effect on which value I choose for the UUID. But I can change it to be unique just to be sure.